packages/angular-cli/blueprints/ng2/files/package.json

```diff
@@ -41,7 +40,6 @@
     "karma-remap-istanbul": "^0.2.1",
     "protractor": "~4.0.13",
     "ts-node": "1.2.1",
-    "tslint": "^4.3.0",
```

Comment:
@hansl please let me know if you want this line back in. It's technically not necessary anymore since it's using the CLI `tslint` dependency and not the project's dependency.